### DISCUSSION TITLE
Why use containers instead of  initContainers in the section of  ”Setup in Kubernetes“ ？

### DISCUSSION THREAD

**Discussion by hardy4yooz** [apache/skywalking#7867] [General]
https://skywalking.apache.org/docs/skywalking-java/latest/en/setup/service-agent/java-agent/containerization/#kubernetes

This will cause the pod status to be `NotReady`.

**Comment by kezhenxu94** [comment]
@hardy4yooz should be init container, can you open a pull request to update?

**Reply by hardy4yooz** [reply]
https://github.com/apache/skywalking-java/pull/44
